### ISSUE TITLE
Binary image for measuring accruacy

### ISSUE THREAD

**Issue by xinosaurous** [MaybeShewill-CV/lanenet-lane-detection#372]
Hi, 

I think I'm not understanding how the accuracy calculation works. 
Looking at the code, it looks like the post processing is only applied in the testing step. Since the accuracy is calculated on the binary image during only does that mean the post processing is not affecting the accuracy measurement? Doesn't that make the image less accurate?

**Comment by MaybeShewill-CV** [comment]
@xinosaurous You may check this https://github.com/MaybeShewill-CV/lanenet-lane-detection/blob/master/tools/evaluate_model_utils.py

**Comment by xinosaurous** [comment]
Yes, I have looked at that. I'm wondering why the script passes in the binary segmentation logits instead of the image after post-processing. Like, the binary segmentation hasn't gone through the clustering or curve fitting so it would be less accurate right?

**Comment by MaybeShewill-CV** [comment]
@xinosaurous The precision only represent the model's segmentation performance. If you want to evaluate the lane detection precision you may get some help by checking this https://github.com/TuSimple/tusimple-benchmark

**Comment by xinosaurous** [comment]
I see, thank you for your help.

**Comment by MaybeShewill-CV** [comment]
@xinosaurous Welcome:)
